.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,19 +1,9 @@
-----
 <!-- Thank you for sending a PR! -->
-<!-- Please perform the following checks and check all the boxes that apply. -->
-<!-- If your PR does not create a command page,
-     you can remove the first two checklist items. -->
-<!-- If your PR neither creates nor edits a command page (e.g. README edits, etc.)
-     you can simply remove the entire checklist. -->
+<!-- Please perform the following checks and mark all the boxes accordingly. -->
+<!-- You can remove the checklist items that don't apply to your PR. -->
 
 - [ ] The page (if new), does not already exist in the repo.
-
-- [ ] The page (if new), has been added to the correct platform folder:  
-      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.
-
+- [ ] The page is in the correct platform folder (`common/`, `linux/`, etc.)
 - [ ] The page has 8 or fewer examples.
-
-- [ ] The PR is appropriately titled:  
-      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.
-
-- [ ] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.
+- [ ] The PR title conforms to the recommended [templates](CONTRIBUTING.md#commit-message).
```

Comment:
template ?

Reply:
My thinking was that there are several formats, depending on the type of change. I considered each of them a template for the commit message and PR title for that kind of change.

Reply:
This will display as markdown, right? Just curious if the links will show up. Even if not, I think it should be easy enough to work out which file & section it's referring to.

Reply:
It will display as markdown.

Just wondering though if using `CONTRIBUTING.md` is right. I would have thought that it would point to `https://github.com/tldr-pages/tldr/pull/CONTRIBUTING.md`?

Reply:
Indeed it probably will, unless GH does some mangling. This is a test: [test](CONTRIBUTING.md)

Reply:
Nope, no mangling here.

Reply:
Maybe it's better to do this?
`https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message`

Reply:
It will display as markdown, but only in the preview mode when composing the PR message, or after saving the PR.

The reasoning for the short link is to make the template as compact as possible, in order not to overwhelm contributors when they're creating a PR.

In any case, relative links are a feature of markdown, so even though it would indeed be more explicit to include the full URL, it's not really wrong to use the short link, if that makes the template cleaner and more likely to be read in full.

That said, I just noticed, thanks to your conversation, that the relative link from a PR page does not produce the intended URL. Writing `[foo](CONTRIBUTING.md)` produces a link pointing to

>  https://github.com/tldr-pages/tldr/pull/CONTRIBUTING.md

rather than 

>  https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md

So we'll have to tweak the relative link, like so: `[foo](../blob/master/CONTRIBUTING.md)`. I'll make the change now.

Reply:
Yeah, I was originally going to do that, but realised that when you go to make a PR it's from a different url. i.e. `...tldr-pages/tldr/compare/master...{user}:{branch}`

If using the relative one, then when they are writing their PR and filling in the details, it will actually point to: `...tldr-pages/tldr/compare/blob/master/CONTRIBUTING.md`

Reply:
You can prefix it with a `/` to make a url relative to the current domain, if that makes it any more concise.

Reply:
@pxgamer I just tested your hypothesis to be sure. The `../` in the relative link URL works just like in unix paths, i.e. it moves one level up, and that applies to both the URL where PRs are created from (`...tldr-pages/tldr/compare/...`) and to the one where the PR lives after it's created (`...tldr-pages/tldr/pull/...`).

So in this case, `[foo](../blob/master/CONTRIBUTING.md)` does indeed work as intended, producing an URL to

    https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md

So this PR can be merged as it is, after all :)

Reply:
I realised why it was failing for me, because I use GitFlow formatting for my branches. So they contain a `/` (e.g. `feature/command`)

So as long as people don't use slashes in branch names (which most newcomers probably wont) it should be fine. :+1:

Reply:
Hmmm, that's an interesting edge case. It will fail then, indeed. Perhaps to keep it safe we should use the trick @sbrl mentioned then? i.e.

    [foo](/tldr-pages/tldr/blob/master/CONTRIBUTING.md)

which at least is shorter than the full-fledged

    [foo](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md)

WDYT?

Reply:
D'oh. Merged to quickly. :yum:

I think that sounds good. :+1:

Reply:
Great! I'll submit a new PR.